src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -911,7 +911,7 @@ export default {
           filingType,
           title,
           filingId: header.filingId,
-          filingAuthor: null,
+          filingAuthor: this.filingAuthor(filing),
```

Comment:
The API will now control what is displayed for Colin filings (if anything -- this is backward compatible).